### ISSUE TITLE
Sugest

### ISSUE THREAD

**Issue by devv4646456** [zDevelopers/ImageOnMap#139]
It would be nice if there was an option to not be able to put up an image so quickly. Some kind of minimal delay

**Comment by Vlammar** [comment]
> 
> 
> It would be nice if there was an option to not be able to put up an image so quickly. Some kind of minimal delay

Why for ? I don't see any usage for this feature. Can you detail a bit more ?

**Comment by devv4646456** [comment]
> Why for ? I don't see any usage for this feature. Can you detail a bit more ?

Of course. Larger image sizes cause fps and even tps to drop to 7. If the map is 20x20 and a player wants to harm the server it is trivial for him.

**Comment by devv4646456** [comment]
I wanted to give players several of these 10x10 maps, but I'm afraid it will cause huge problems when they start doing it at the same time. One 20x20 map will cause the tps to drop to 7. That's what I checked
